core/lib/l1_contract_interface/src/i_executor/methods/commit_batches.rs

```diff
@@ -24,3 +27,27 @@ impl Tokenize for CommitBatches {
         vec![stored_batch_info, Token::Array(l1_batches_to_commit)]
     }
 }
+
+impl CommitBatches {
+    fn into_tokens_blobs(self, number_of_blobs: usize) -> Vec<Token> {
```

Comment:
I suggest removing `into_tokens_blobs` and `into_tokens_calldata`. Instead you can add mode encoding param to `CommitBatches` and use it in `into_tokens`. This way code will be cleaner